### DISCUSSION TITLE
New Feature: Auto-Unmonitor NotFound albums?

### DISCUSSION THREAD

**Discussion by RandomNinjaAtk** [RandomNinjaAtk/docker-lidarr-extended#24] [Polls]
So albums that are notfound are currently skipped using a log file, which facilitates faster processing on future script runs. However, I just had a thought that I could probably implement a way for the script automatically unmonitor the albums in Lidarr, thus removing them from the list in an automated way and speeding up future runs and processing.

**Comment by RandomUser345** [comment]
Sounds nice, only two things to consider: what if the album is very new and not yet available and what if it's just temporarly not available and will show up in the future.
I guess for the second point there is no real workaround, but the first can be mitigated with checking the releasedate / status and not unmonitor if lets say releasedate + 1month < today

**Comment by kingofdunghill** [comment]
Good idea. Since there is no possibility to define the search interval, my cpu is enormously stressed by the 15 minute scans with several hundreds of missing albums.

But not monitoring them also leads to the fact that they can no longer be found via torrent / usenet indexer.

**Reply by RandomNinjaAtk** [reply]
It's not documented, but you could set this docker parameter "scriptInterval" to adjust it, just like you could on the previous AMD project. 

It uses the sleep command, so keep in mind you need to set it to something compatible:
https://github.com/RandomNinjaAtk/docker-lidarr-extended/blob/5b1858caae88109de8bd5c17784c3740182a8394/root/scripts/start.sh#L29

**Comment by Protektor-Desura** [comment]
I think if you change, if Lidarr keeps looking for an album that is bad. Because for me I use your scripts and torrent sites to keep looking for albums. If you change it to unmonitored then it won't check torrents or usenet if someone uploads the album or someone requests it on a torrent site/usenet then it won't be picked up and auto downloaded.

Instead it should behave like AMD did. It searched for it, couldn't find it so it won't try again to download it.

**Comment by RandomNinjaAtk** [comment]
I think I found a better way to handle this, so this is likely not needed anymore. The latest script changes solves this by only re-searching notfound albums once every 7 days... 

It might be worth considering to up that number to once a month though...

Edit: The script already did this, but I eliminated processing them twice by skipping notfound during the missing/cutoff list generation process. Making subsequent runs nearly instant, if nothing had changed between them...
